### ISSUE TITLE
Installation Guide - Link to STMS empty - SAPLink nuggets appear old

### ISSUE THREAD

**Issue by RainerWinkler** [abap2xlsx/abap2xlsx#523]
Dear colleagues,

in the wiki it is told that there is beside of abapGit (Preferred) also the option to install via STMS or SAPLink. But the link behind STMS appears to be empty. And when I look for SAPLink Nuggets in the given links, I see only old nuggets.

Do I miss some information?

Or should we remove the links to STMS and SAPLink as it appears that abapGit is the only pratical path to install abap2xlsx.

Thank you very much,
Rainer Winkler

**Comment by ivanfemia** [comment]
Rainer,
you are right, STMS and SAPLink are obsolete and no longer maintained.

**Comment by gregorwolf** [comment]
Dear Rainer,
what you could do is to import abap2xlsx in your NetWeaver ABAP Developer Edition using abapGit and assign the Objects to a package that is assigned to a transport request. That way you can create a transport request containing the latest abap2xlsx version. In future we might be able to use ABAP on SAP Cloud Platform to automate this process.
Best regards
Gregor

**Comment by RainerWinkler** [comment]
Dear Ivan, Dear Gregor,
thank you very much. I will see whether I manage to get a transport file from my developer edition.
Thank you very much,
Rainer

**Comment by RainerWinkler** [comment]
Dear Ivan, Dear Gregor,
I used now the export and import to and from zip file function from abapGit. I exported from my developer edition and imported into a sandbox system of my customer. This worked well. Although I was forced to repeat the import a few times (thereby sometimes overwriting "locally" changed objects or not setting the overwriting). After a few tries I was able to run ZDEMO_EXCEL without errors.
Thank you very much 
Rainer
